packages/devtools_server/pubspec.yaml

```diff
@@ -17,5 +17,5 @@ dependencies:
   pedantic: ^1.7.0
   shelf: ^0.7.4
   shelf_static: ^0.2.8
-  vm_service_lib: 3.21.0
+  vm_service_lib: 3.22.0
```

Comment:
I believe that rolling this forward will cause NPEs in parts of the app (I did some spot checking of the memory page). We may want to de-couple rolling to this version with the support for the `_registerService` rename until we've had time to investigate any NPEs.

Reply:
Ah, nevermind this - I'd through this change was in packages/devtools/pubspec.yaml.